sdk/websites/Azure.ResourceManager.AppService/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
```

Comment:
specify the release date if you want to release it after this PR

Reply:
Release data has been added

Reply:
Fixed.